Sources/Core/JavaFileRenderer.swift

```diff
@@ -90,7 +90,7 @@ extension JavaFileRenderer {
                 fatalError("Bad reference found in schema for class: \(className)")
             }
         case .oneOf(types: _):
-            return "\(className)\(param.snakeCaseToCamelCase())"
+            return "\(className)\(Languages.java.snakeCaseToCamelCase(param))"
```

Comment:
Can you file a ticket for abstracting the language type here ? I think the context of java / objc render should be able to influence which `snakeCaseToCamelCase` logic is used

Reply:
https://github.com/pinterest/plank/issues/190